op-node/rollup/derive/system_config_test.go

```diff
@@ -138,6 +138,33 @@ func TestProcessSystemConfigUpdateLogEvent(t *testing.T) {
 			},
 			err: false,
 		},
+		{
+			// The ecotone scalars should be updated
+			name: "SystemConfigUpdateGasConfigEcotone",
+			log: &types.Log{
+				Topics: []common.Hash{
+					ConfigUpdateEventABIHash,
+					ConfigUpdateEventVersion0,
+					SystemConfigUpdateGasConfigEcotone,
+				},
+			},
+			hook: func(t *testing.T, log *types.Log) *types.Log {
+				basefeeScalar := big.NewInt(0xaa)
+				blobBasefeeScalar := big.NewInt(0xbb)
+				packed := make([]byte, 8)
+				basefeeScalar.FillBytes(packed[0:4])
+				blobBasefeeScalar.FillBytes(packed[4:8])
+				data, err := bytesArgs.Pack(packed)
+				require.NoError(t, err)
+				log.Data = data
```

Comment:
This isn't technically tied to the way that the log is actually serialized by the evm. I would need to go in and confirm using `chisel` or a solidity script that uses the get logs cheatcode to observe an actual log to be certain that this serialization is correct. The other tests here aren't ideal with this either

Reply:
is there something I can do in this PR to address or can we leave it to follow-ups?

Reply:
We can address as a follow up to resolve #8801